### PR TITLE
5.0.x

### PR DESCRIPTION
**Pull request by ustmaestro**
Hello!

* Type: new feature
* Link to issue: https://github.com/phalcon/cphalcon/issues/15919

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [x] I wrote some tests for this PR
- [ ] I have updated the relevant CHANGELOG
- [ ] I have created a PR for the [documentation](https://github.com/phalcon/docs) about this change

Small description of change:

Added support to class constants annotations

```php
    class MyClass
    {
          /**
           * @Simple
           */
          const MY_CONST = "test";
    }
    
    
    use \Phalcon\Annotations\Adapter\Memory;
    use \Phalcon\Annotations\Reader;
    use \Phalcon\Annotations\Reflection;  
    
    // Reader now returns constants array list that have annotations in docblock comments
    $reader = new Reader();
    $data = $reader->parse(MyClass::class);
    var_dump($data);

    // Reflection has a new method for getting constants annotations
    $reflection = new Reflection($data);
    var_dump($reflection->getConstantsAnnotations());


    // Annotation adapters has 2 new methods for getting constants annotations
    $memory = new Memory();
    var_dump($memory->getConstants(MyClass::class));                  // returns array
    var_dump($memory->getConstant(MyClass::class, 'TEST2'));     // returns Collection
```

Thanks

### CODE REVIEW

File: phalcon/Annotations/Reader.zep [hunk 1]
```diff
@@ -54,6 +54,46 @@ class Reader implements ReaderInterface
...
+        let constants = reflection->getConstants();
+
+        if count(constants) {
+            /**
+             * Line declaration for constants isn't available
+             */
+            let line = 1;
+            let anotationsConstants = [];
+
+            for constant in array_keys(constants) {
```
Comment (niden) [inline phalcon/Annotations/Reader.zep:69]: Add a variable and assign `array_keys(constants` to it.

**Review by niden — CHANGES_REQUESTED** [review]
Please update the interface as well with the two new methods. Also we need the CHANGELOG to be updated.


Great work!!

**Comment by niden** [comment]
@ustmaestro Just update the changelog and this is ready!!

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @ustmaestro !

**Comment by okool** [comment]
Thank @ustmaestro

**Pull request merged**